source/extensions/quic_listeners/quiche/BUILD

```diff
@@ -369,6 +369,7 @@ envoy_cc_library(
 # All of these are needed for this extension to function.
 envoy_cc_extension(
     name = "quic_factory_lib",
+    category = "SOMECAT",
```

Comment:
i see

```
$ grep quic catlist.txt 
envoy.quic_client_codec
envoy.quic_server_codec
```
so im guessing `envoy.quic_server_codec`

Reply:
I think this is transport sockets actually. You can get a clue by looking at source/extensions/extensions_build_config.bzl.

Reply:
ah k - that changes some of them - ill go through again